src/types/decimal.rs

```diff
@@ -29,20 +30,18 @@ impl Decimal {
     }
 
     /// Returns scale of the Decimal value
-    /// A scale indicates the number of digits to the right of the decimal point.
+    /// If zero or positive, a scale indicates the number of digits to the right of the decimal point.
+    /// If negative, the unscaled value of the number is multiplied by ten to the power of the negation of the scale. For example, a scale of -3 means the unscaled value is multiplied by 1000.
```

Comment:
Nit: I suggest wrapping this line.
```suggestion
    /// If negative, the unscaled value of the number is multiplied by ten to the power of the negation of the scale.
    /// For example, a scale of -3 means the unscaled value is multiplied by 1000.
```